### ISSUE TITLE
How can participants find private projects? 

### ISSUE THREAD

**Issue by phillimorland** [liqd/adhocracy-plus#1790]
This is more a question to the PM's, @lthueer @fredewe I was doing some testing due to comment info message being wrong for private projects and semi private projects and found that there does not seem to be a way for a user to find a private project from their profile as the activity is not listed, is this how it should be?

**Comment by lthueer** [comment]
@phillimorland you're right, there is no way for finding a private project in the profile (following/activities). That's why it was developed last year that an email is send after the invitation to a private project is accepted which contains the link to the project. I think it would be nice to enhace this at some point.

**Comment by fredewe** [comment]
@phillimorland I would write a user story and delete this issue, ok?

**Comment by fredewe** [comment]
You can find the user story here: 
https://taiga.liqd.net/project/liqd-liquid-product/us/251?no-milestone=1
